### DISCUSSION TITLE
Long titles going offscreen

### DISCUSSION THREAD

**Discussion by KD-7** [pradyunsg/furo#640] [Potential Issue]
Hi,

Im facing issues with long titles not wrapping around the screen:

![image](https://user-images.githubusercontent.com/91730569/226490983-dc31a111-821d-4b6e-9ab5-1de71168b5b3.png)

I was wondering in this case if it was possible to dynamically resize the header or wrap it around. This is happening when the title has no spaces.

**Comment by pradyunsg** [comment]
https://pradyunsg.me/furo/kitchen-sink/really-long/#heading-14-is-a-really-long-heading-that-is-also-code-and-should-wrap-when-it-reaches-the-sidebar

This works just fine.
